frontend/views/containers/sidebar/GroupsList.vue

```diff
@@ -21,8 +21,11 @@ ul.c-group-list(v-if='groupsByName.length')
       direction='right'
       :text='L("Create a new group")'
     )
-      router-link.button.is-icon.has-background(
-        to='/new-group/name'
+      button(
+        class='is-icon has-background'
+        @click='openModal("CreateGroup")'
+        data-test='createGroup'
+        alt='L("Add a group")'
```

Comment:
Missing `:`, and not sure what the difference is between `:aria-label` and `:alt`... which should be used here?

cc @sandrina-p